src/podbench/execfile.py

```
@@ -0,0 +1,179 @@
+from pathlib import Path, PurePosixPath
+
+from .proc import DEFAULT_PROC, same_root, strip_deleted, sysroot_path
+
+__all__ = [
+    "DEFAULT_STAGING",
+    "gdb_exec_file",
+    "shadowing_file",
+    "stage_exec_file",
+]
+
+DEFAULT_STAGING = Path("/tmp/podbench-exe")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/podbench-exe"

(S108)